src/MLHeader/MLHeader.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { PageHeader } from 'antd'
+import './style'
+
+const MLHeader = (props) => {
+  const avatarProps = {size: 48, ...props.avatar}
```

Comment:
@phoenixeliot Looking pretty good now except the Avatar size isn't passing through at the React level and I don't understand why.

Reply:
Fixed; this is because `{...props}` was overwriting the `avatar=...` above. When a prop is set twice, the last one is the one that takes effect.